src/proto/thread_telemetry.proto

```diff
@@ -513,6 +513,69 @@ message TelemetryData {
     optional bool has_others_route_added = 3;
   }
 
+  enum EpskcState {
+    EPSKC_STATE_READY_FOR_CONNECTION = 0;
```

Comment:
Normally the 0 value should be unspecified.  https://google.aip.dev/126

Reply:
Done